addons/barcodes/models/barcode_nomenclature.py

```diff
@@ -16,7 +16,7 @@ class BarcodeNomenclature(models.Model):
     _name = 'barcode.nomenclature'
     _description = 'Barcode Nomenclature'
 
-    name = fields.Char(string='Barcode Nomenclature', size=32, required=True, help='An internal identification of the barcode nomenclature')
+    name = fields.Char(string='Barcode Nomenclature', required=True, help='An internal identification of the barcode nomenclature')
```

Comment:
The commit message is useless. You should rather explain why we made the change. In this case, it should be
`Due to GS1, now the barcode contains a lot more than a single identifier for and object (e.g. product, location,...). Sometimes it contains a combination of multiple informations. It's common now to have a size greater than 32 that why we remove the parameter.`